content/blog/2021/07/camel-jbang/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Camel + JBang: bringing the world of Java scripting to Camel"
+date: 2021-07-26
+authors: ["orpiske"]
+categories: ["Features", "Camel"]
+preview: "Camel + JBang"
+summary: "Bringing the world of Java scripting to Camel"
+---
+
+One of the new features we are introducing with Camel 3.12 is a command-line utility that leverages [JBang](https://www.jbang.dev/) in order to make it possible to benefit from the large set of Apache Camel features, without the need of writing a full blown application.
+
+Before showing what it can do, it's important to give a brief overview of what JBang is. According to their website, JBang lets us " … create, edit and run self-contained source-only Java programs with unprecedented ease". In practice, this means that it lets us run Java applications without the need to write an application, going through the usual build/compile/deploy cycle. It turns Java into a scripting language, in a similar fashion to Python, Shell Script and others. With easy access to the dependency management, configuration, debugging and other features, it makes it very easy for Java developers to use all the power that Java and its ecosystem of components can offer.
+
+We are leveraging the set of features provided by JBang to expose and simplify access for some of the Apache Camel features with the intention to simplify trying Camel; make it easier to learn about and search for components, languages and features; and provide mechanisms for the users to try agile integration.
+
+
+To showcase the features that have been already implemented, this we prepared a set of screencasts showing the installation process, the
```

Comment:
The "this" in the middle sentence is not needed I think.

Reply:
Indeed!